sfge/src/main/java/com/salesforce/graph/ops/registry/Registry.java

```diff
@@ -0,0 +1,97 @@
+package com.salesforce.graph.ops.registry;
+
+import com.salesforce.graph.ops.expander.PathExpansionRegistry;
+import com.salesforce.graph.ops.expander.PathExpansionRegistryUtil;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Supplier;
+
+/**
+ * Registry framework can be used as an optimization mechanism to store smaller data in collections.
```

Comment:
Can you add linebreaks for the documentation here? Also, should all of these things be done at once, or is it a "pick one" situation?

Reply:
Ah, that's google formatter. Will add these as line items.
All things need to be done serially.